linera-version/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "linera-version"
+version = "0.8.0"
+authors = ["Linera <contact@linera.io>"]
+readme = "README.md"
+repository = "https://github.com/linera-io/linera-protocol"
+homepage = "https://linera.dev"
+documentation = "https://docs.rs/linera-rpc/latest/linera_rpc/"
+license = "Apache-2.0"
+edition = "2021"
+
+[dependencies]
+async-graphql.workspace = true
```

Comment:
Is this the same as `async-graphql = { workpace = true }` ?

Reply:
Yep, just a bit cleaner.